src/main/java/com/example/Library/security/JwtTokenProvider.java

```diff
@@ -59,6 +61,7 @@ public boolean validateToken(String token) {
             Jws<Claims> claimsJws = Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token);
             return !claimsJws.getBody().getExpiration().before(new Date());
         } catch (JwtException | IllegalArgumentException e) {
+            log.error("Jwt token is expired or invalid. HttpStatus 401");
```

Comment:
if you need to print a stack trace via a logger, you can use overloaded version of the same *.error() method accepting as input the exception itself:
logger.error("An exception occurred!", <some exception>);